python/FlexSource.py

```diff
@@ -123,12 +140,13 @@ def start(self):
         for p in pans:
             p.Close(True)
         self.pan_adapter = self.radio.GetOrCreatePanadapterSync(0, 0)
-        #self.pan_adapter.PropertyChanged += self.__property_changed
+        # self.pan_adapter.PropertyChanged += self.__property_changed
```

Comment:
We would probably get rid of this now that we're done debugging things

Reply:
I still use it a bunch when testing new pieces. I'll tag it for removal when I haven't used it in a bit or have a different testing system setup.